manticore/core/cpu/arm.py

```diff
@@ -752,7 +752,6 @@ def UXTB(cpu, dest, src):
     @instruction
     def PLD(cpu, addr, offset=None):
         """PLD instructs the cpu that the address at addr might be loaded soon."""
-        pass
```

Comment:
Heh, actually having `pass` in an empty functon or empty exception class is the way to go. I wonder why autoflake deletes it :(

Reply:
Way to go in what sense?

Reply:
I mean it is an explicit way to say "yeah I really did wanted this code block to be empty".

Some links:
* https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions - e.g. the examples for empty user-defined exceptions use `pass`
* https://docs.python.org/3.2/reference/simple_stmts.html#the-pass-statement

Reply:
I somewhat prefer having the pass, but it is redundant if the docstring is present.

Reply:
That said, I don't really think it's worth going through the work to add them all back in, if it's not as simple as a git revert.